pyproject.toml

```diff
@@ -43,7 +43,7 @@ websockets = "^12.0"
 clickhouse-connect = "^0.7.16"
 kafka-python = "^2.0.2"
 jwcrypto = "^1.5.6"
-h2 = "^4.1.0"
+h2 = {git = "https://github.com/python-hyper/h2"}
```

Comment:
Let's pin it to a commit to make it more cache-friendly and reproducible?

Reply:
I assumed the lockfile would cover this

Reply:
You're right! 
I haven't noticed thanes in `poetry.lock` at first (and was surprised by it)